Instructions on how to remove inductiva (#1091)
inductiva/inductiva: docs/overview/uninstall_inductiva.md

# Uninstall the `inductiva` package

> Whether you just want a completely new installation of `inductiva` or
> whether you decided that, as it turns out, the package is not for you,
> we understand and would like to thank you for having given it a
> go. Either way we would like to give you instructions on how to
> completely remove traces of `inductiva` from your system.

Reviewer: ```suggestion
Whether you just want a completely new installation of `inductiva` or
simply want to completely remove the package from your system, we've got
you covered. Either way, we would like to give you instructions on how to
completely remove any trace of `inductiva` from your system.

**NOTE** No simulation files generated though the package will be removed.
```

> First you should remove it using:

Reviewer: ```suggestion
First, you should uninstall the packege the using:
```

Reviewer 2: *package :D

```shell
pip uninstall inductiva
```

Next, recall that the inductiva package, writes locally to your

> disc. Therefore you should have a folder under `$HOME/.inductiva` in
> your system. Remove it. You are good to go!

Reviewer: ````suggestion
disc. Therefore, you should have a folder under `$HOME/.inductiva` in
your system. Remove it and you are good to go!

```shell
rm -rf $HOME/.inductiva
```
````

Reviewer 2: Use *disk instead, since that refers more to a computational sense and what we use as arg in the library.
